src/server.ts

```diff
@@ -10,6 +13,21 @@ function getRovers(res: any) {
       .then((e) => res.send(e.data));
   }
 
+  function getPhotos(rovername: rover, cameratype: camera, res: any) {

```

Comment:
To avoid needing to use the `any` type for Express, you can install the express types with `npm install --save-dev @types/express`

Modules were originally built for JS (without types) but once TS got popularity, people created @types modules for you to add as dev dependencies, giving you access to type definitions for existing JS modules